client/src/components/BottomPanel.tsx

```diff
@@ -76,6 +81,29 @@ export const BottomPanel = ({ isOpen, type, onAction }: IPanelProps) => {
 						}}
 					/>
 				);
+			case 'background':
+				return (
+					<>
+						{images.map(({ key, src }) => (
+							<div key={key} className="bottom-panel-background">
+								<IconButton onClick={() => {
+									clearTimeout(timers[0]); // clear any existing timers
+									onAction('background', key); //send background image to app.tsx
+									currentTimer = window.setTimeout(() => { // get current timer id
+										console.log('in setTimeout');
+										onAction('background', 'undefined'); // send undefined image to app.tsx after 1 minute
```

Comment:
should use an undefined variable, we want to minimize cases like 'true', 'undefined', because they aren't the actual values of true, undefined, etc and can cause unclear code

if you do onAction('background') then the second parameter is automatically undefined, or you could do onAction('background', undefined) without quotation marks

Reply:
I attempted to use undefined without quotation marks, but it did not reset the background when sent. I will try with just onAction('background')